app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:id])
```

Comment:
- [ ] We should use `params[:post_id]` here to be more specific in the id we want to use.